app/serializers/v1/legacy/event_error_serializer.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module V1
+  module Legacy
+    class EventErrorSerializer < ModelSerializer
+      def serialize
+        {
+          input_params: {
+            transaction_id: model.transaction_id,
+            external_subscription_id: model.external_subscription_id,
+            external_customer_id: model.external_customer_id,
+            timestamp: model.timestamp.to_f,
+            code: model.code,
+            properties: model.properties,
+          }
```

Comment:
Style/TrailingCommaInHashLiteral: Put a comma after the last item of a multiline hash.